lib/stores/filesystem.js

```diff
@@ -305,6 +326,35 @@ class FilesystemStore {
     return { size, md5 };
   }
 
+  async putObjectTagging(bucket, key, content) {
+    const taggingPath = this.getResourcePath(bucket, key, "tagging.json");
+    const parsedTagging = await parseXmlBody(content);
```

Comment:
@kherock Not sure what's the preferred approach here. The SDK sends tags as XML in the request body, I parse this and then save it to JSON, and then on the way out I do the reverse. I'm wondering if I should just take the payload, save it, and then send it back out again for Get requests without touching it at all?

Reply:
That's the approach I would have tried first - you should try seeing if you can reuse `pubSubresource` and `getSubresource` for this. However, those methods are defined to use a config model (such as ones we have for website and CORS configurations) that would parse and validate tags. Let me know if you need more guidance on that.